### ISSUE TITLE
Coverall job is stuck and my GitHub PR is blocked

### ISSUE THREAD

**Issue by GeorgiStavrev** [lemurheavy/coveralls-public#1519]
Hi,

I have a Coveralls job https://coveralls.io/jobs/74251251 which is not reporting any progress and this blocks one of my PRs on a private repo. This is my 2nd PR, I had this issue with the first, so I closed it and created a new PR to try to unblock myself.

There's an alert poping up when I go to the job page saying: "DataTables warning: table id=DataTables_Table_0 - Ajax error. For more information about this error, please see http://datatables.net/tn/7"

**Comment by afinetooth** [comment]
Hi @GeorgiStavrev,

Yes, I can see the fall off in your build history here:
https://coveralls.io/repos/322673/builds

Toward the end of the first page, where coverage starts reporting as blank/empty:

![Screen Shot 2021-01-29 at 11 23 04 AM](https://user-images.githubusercontent.com/918667/106318176-5b988900-6224-11eb-8677-bcd551b9b238.png)

Looking at the build pages for the last good and first bad builds there, we can see the main difference being the number of files processed:

https://coveralls.io/builds/36694218

![Screen Shot 2021-01-29 at 11 23 53 AM](https://user-images.githubusercontent.com/918667/106318240-78cd5780-6224-11eb-8e59-37ea2b42f992.png)

(1874 files)

Vs:

https://coveralls.io/builds/36695314

![Screen Shot 2021-01-29 at 11 24 29 AM](https://user-images.githubusercontent.com/918667/106318289-8e428180-6224-11eb-8704-c6f7235ab78e.png)

(0 files)

That indicates the background jobs responsible for calculating your builds' coverage, and drawing your directory tree (the DataTables warning is a JS package that renders the tree view in the Source Files table), have not completed.

This tracks with a performance issue we've been seeing on some projects for the past 48-72 hrs, particularly those with a number of files north of 1,000.

We're aware of the issue and are working on it now. I'll feedback when I have any change in status.

In the meantime, rest assured, those jobs will be retried and in 98% cases they will complete and your build will be fine. It's just a matter of waiting right now until we can resolve. 

Thanks for the report.

You can keep updated on status here:
https://status.coveralls.io/

**Comment by afinetooth** [comment]
Hi @GeorgiStavrev, the underlying performance issue should be resolved now.

I can see that all of your builds are now complete. Let me know if you see anything else.

**Comment by GeorgiStavrev** [comment]
It looks like everything is fine for us right now. Thanks @afinetooth

**Comment by afinetooth** [comment]
@GeorgiStavrev glad to hear it. You’re welcome.
